From ea59b26135c3cd4cc67f43d566b9d1ca312d0957 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Fri, 26 May 2017 16:12:26 -0800
Subject: [PATCH 2486/2748] Revert "HADOOP-14100. Upgrade Jsch jar to latest
 version to fix vulnerability in old versions.
 Contributed by Vinayakumar B and Brahma Reddy
 Battula."

This reverts commit acb1a8a3b99cbee85b84b87ef9261ccf95c25b5e.

https://jira.cloudera.com/browse/CDH-54459 Post-commit failure due to HADOOP-14100 Upgrade Jsch jar

Change-Id: I2b447f52a027c2e0373ea2649d8f884f1d045300
---
 hadoop-project/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 2d80f1d..750099d 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -819,7 +819,7 @@
       <dependency>
         <groupId>com.jcraft</groupId>
         <artifactId>jsch</artifactId>
-        <version>0.1.54</version>
+        <version>0.1.42</version>
       </dependency>
       <dependency>
         <groupId>org.apache.htrace</groupId>
-- 
1.7.9.5

